src/views/add-collaborator.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.html" %}
+
+{% block beforeContent %}
+  {% include "include/back-link.html" %}
```

Comment:
Now that there is a `back-link` block after implementing the alpha phase banner, remember to update this so the back-link isn't overwritten by the phase banner